helpers/build_packages.sh

```diff
@@ -49,6 +49,8 @@ Usage: $0 [OPTION]..."
    -D, --do-not-install
                    useful when package is needed only in the final image
                    especially when it conflicts in an SDK target
+   -N  --no-auto-version
+                   Tell scratchbox to not fix the version inside a spec file
```

Comment:
This and other places where this is mentioned should have mb2 instead of scratchbox.

Reply:
Done.